outputs.tf

```diff
@@ -61,6 +61,11 @@ output "cloudwatch_log_group_name" {
   value       = aws_cloudwatch_log_group.this[*].name
 }
 
+output "cloudwatch_log_group_arn" {
+  description = "Arn of cloudwatch log group created"
+  value       = aws_cloudwatch_log_group.this[*].arn
```

Comment:
Can you please return a string instead of a list please. See https://github.com/terraform-aws-modules/terraform-aws-eks/pull/1061

Reply:
thanks. updated https://github.com/terraform-aws-modules/terraform-aws-eks/pull/1071/commits/5f61c20b4f03944f04b50d7add5c0f0caa5a72c1